test/test_playbooks/tasks/product.yml

```diff
@@ -4,6 +4,7 @@
     - organization_name: "Test Organization"
     - product_name: "Test Product"
     - product_description: "A happy little test product"
+    - product_state: present
```

Comment:
I think, we left that one out for a reason, as we wanted to be specific about the creating/deleting every time using it.
@akofink , since introduced these tasks, i guess it's your call.

Reply:
I don't feel strongly about it, but no other tasks provide the state as a default.

Reply:
It felt easier to override with more explicit testing on state if its defined with a default than if the parameter is left off.

Reply:
Ok for me. It does not hurt anyway.